terraform/gcp/main.tf

```diff
@@ -27,13 +27,13 @@ provider "google" {
 
 data "google_client_config" "provider" {}
 
```

Comment:
why is this commented out?

Reply:
Oops - forgot to clean out commented code. Originally, I was creating the kubernetes service account in terraform, but this meant that in the same module terraform was creating a k8s cluster and also connecting to it and creating things on it. This is generally not recommended to do because terraform can drift really easily and get confused. I didn't know this until I started running into issues connecting to the cluster. But since the service account is just a standard k8s manifest, I just moved it to a normal yaml file that we create manually (like all other k8s manifests) instead of creating it here.